src/components/Garden/DecisionDetail/VoteStatus.js

```diff
@@ -2,8 +2,8 @@ import React from 'react'
 import styled from 'styled-components'
 import { IconCheck, IconCross, GU, textStyle, useTheme } from '@1hive/1hive-ui'
 
-import celesteIconSvg from '../../assets/celeste-icon.svg'
-import challengeIconSvg from '../../assets/challenge-icon.svg'
+import celesteIconSvg from '@/assets/celeste-icon.svg'
+import challengeIconSvg from '@/assets/challenge-icon.svg'
```

Comment:
Event hough is the same, should we use the direct alias?

```suggestion
import celesteIconSvg from '@assets/celeste-icon.svg'
import challengeIconSvg from '@assets/challenge-icon.svg'
```